test/xpu_api/numerics/complex.number/cmplx.over/pow.pass.cpp

```diff
@@ -84,26 +84,18 @@ ONEDPL_TEST_NUM_MAIN
     test<unsigned, float>();
     test<long long, float>();
 
-    auto fnc1 = []()
-    {
-        test<int, double>();
-        test<unsigned, double>();
-        test<long long, double>();
-        test<float, double>();
-        test<double, float>();
-    };
-    IF_DOUBLE_SUPPORT(fnc1);
-
-    auto fnc2 = []()
-    {
-        test<unsigned, long double>();
-        test<long long, long double>();
-        test<float, long double>();
-        test<double, long double>();
-        test<long double, float>();
-        test<long double, double>();
-    };
-    IF_LONG_DOUBLE_SUPPORT(fnc2);
+    IF_DOUBLE_SUPPORT(test<int, double>();
+                      test<unsigned, double>();
+                      test<long long, double>();
+                      test<float, double>();
+                      test<double, float>())
+
+    IF_LONG_DOUBLE_SUPPORT(test<unsigned, long double>();
```

Comment:
I made this change to avoid lambdas which are absent in the original test
at https://github.com/intel/llvm/blob/sycl/libcxx/test/std/numerics/complex.number/cmplx.over/pow.pass.cpp